### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by noplanman** [diasporg/Poduptime#101]
Let's try to make podupti.me more accessible!

The main issue is the table of information, which can't be "tabbed" through.
So the only link that gets activated is the pod URL itself. Screen readers then read the title text of the link, which is where the main problem lies at the moment.

We should try to optimise that title text to include just the essential info of a pod, so that blind users can get a quick info of the pod item they're on.

Maybe something like:
`Pod name, Country, # of Users, % Uptime, Signup open`

Since this text gets read for each individual link, it makes sense to remove any duplicate content which is just bulk.

The title as it is now isn't really necessary for sighted users, as all info is visible in the table anyway.

Also, it would be great to have a dedicated filter for certain fields, like Country, Signup, Up-to-date version (or something like that)

@dmorley What are your thoughts on this? Also, try enabling your system screenreader and browsing just by using the keyboard, to get an idea of what it's like.

**Comment by dmorley** [comment]
Interesting screen reader on ubuntu does not even try and read FF or Thunderbird to me. 

Are you saying if you change the table headers from "pod" to "pod name" or the hover data to be less of a mess?

**Comment by noplanman** [comment]
The headers could use some alterations too, but the main thing I'm talking about is the title (popup) of the pod domain links themselves.

![pod-link-title](https://cloud.githubusercontent.com/assets/9423417/22831166/9ecfc61c-efa9-11e6-8529-cd0b4602885e.png)

Listening to each and every entry like this is total overload. I've stayed with a blind couple for a few days last week and they gave me great feedback. Best is as little as possible but quality content, focusing on text instead of images (obviously...).

> Interesting screen reader on ubuntu does not even try and read FF or Thunderbird to me.

Hmm, I'd have thought that it reads anything globally. Maybe a setting somewhere?

**Comment by dmorley** [comment]
Makes sense. Ubuntu is nuts, the screen reader is so fast I can't understand, can't turn on for FF, I searched and found if I install some other packages I might be able to slow down the speech. Anyways Ubuntu Accessibility not the issue. I will try things out on my work machine in a few, I am sure windows does this perfectly!

I am reading around and it seems the alt tag should always be preferred for any html tag, so maybe using alt tags would be a good solution?

**Comment by noplanman** [comment]
> maybe using alt tags would be a good solution

**Absolutely!** You're totally right, it's the `alt` tag that we need to add.
At the same time we can revise the title tag and make it's content better, as it's quite messy at the moment, in my view.

FYI, here's [some reading](https://www.w3.org/WAI/guid-tech) to learn more about accessibility on the web 😋
